### ISSUE TITLE
[Bug] com.intellij.diagnostic.PluginException: Cannot create extension (class=com.github.dinbtechit.vscodetheme.startup.VSCodeStartupNotifyActivity) [Plugin: com.github.dinbtechit.vscodetheme]

### ISSUE THREAD

**Issue by trucuit** [dinbtechit/vscode-theme#130]
#Describe the bug

#To Reproduce

#Expected behavior

#Screenshots

#Environment
* Java: JetBrains s.r.o. 17.0.6
* OS: Mac OS X aarch64
* IDE: WebStorm 2022.3.4
* Version: 1.10.3

#Additional context

#Stacktrace#
 
```
com.intellij.diagnostic.PluginException: Cannot create extension (class=com.github.dinbtechit.vscodetheme.startup.VSCodeStartupNotifyActivity) [Plugin: com.github.dinbtechit.vscodetheme]
	at com.intellij.serviceContainer.ComponentManagerImpl.createError(ComponentManagerImpl.kt:1051)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.doCreateInstance(XmlExtensionAdapter.kt:70)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.createInstance(XmlExtensionAdapter.kt:31)
	at com.intellij.ide.startup.impl.StartupManagerImpl.runPostStartupActivities(StartupManagerImpl.kt:587)
	at com.intellij.ide.startup.impl.StartupManagerImpl.access$runPostStartupActivities(StartupManagerImpl.kt:66)
	at com.intellij.ide.startup.impl.StartupManagerImpl$runStartupActivities$2.invokeSuspend(StartupManagerImpl.kt:184)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:570)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:677)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:664)
Caused by: com.intellij.diagnostic.PluginException: Cannot create class com.github.dinbtechit.vscodetheme.startup.VSCodeStartupNotifyActivity (classloader=PluginClassLoader(plugin=PluginDescriptor(name=VSCode Theme, id=com.github.dinbtechit.vscodetheme, descriptorPath=plugin.xml, path=~/Library/Application Support/JetBrains/WebStorm2022.3/plugins/VSCode Theme, version=1.10.3, package=null, isBundled=false), packagePrefix=null, instanceId=91, state=active))
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClass(ComponentManagerImpl.kt:989)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.instantiateClass(XmlExtensionAdapter.kt:79)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter$SimpleConstructorInjectionAdapter.instantiateClass(XmlExtensionAdapter.kt:90)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.doCreateInstance(XmlExtensionAdapter.kt:50)
	... 10 more
Caused by: java.lang.NoClassDefFoundError: kotlin/enums/EnumEntriesKt
	at com.github.dinbtechit.vscodetheme.startup.VSCodeStartupNotifyActivity$DisplayActionType.<clinit>(VSCodeStartupNotifyActivity.kt:28)
	at com.github.dinbtechit.vscodetheme.startup.VSCodeStartupNotifyActivity.<clinit>(VSCodeStartupNotifyActivity.kt:60)
	at java.base/jdk.internal.misc.Unsafe.allocateInstance(Native Method)
	at java.base/java.lang.invoke.DirectMethodHandle.allocateInstance(DirectMethodHandle.java:520)
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClass(ComponentManagerImpl.kt:950)
	... 13 more
Caused by: java.lang.ClassNotFoundException: kotlin.enums.EnumEntriesKt PluginClassLoader(plugin=PluginDescriptor(name=VSCode Theme, id=com.github.dinbtechit.vscodetheme, descriptorPath=plugin.xml, path=~/Library/Application Support/JetBrains/WebStorm2022.3/plugins/VSCode Theme, version=1.10.3, package=null, isBundled=false), packagePrefix=null, instanceId=91, state=active)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:217)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	... 18 more
```

**Comment by dinbtechit** [comment]
@trucuit  - are you able to upgrade your IDE latest version 2023.2.2? And let me know if this issue still persists.

**Comment by Jev0987** [comment]
thanks, it's useful.

**Comment by dinbtechit** [comment]
Fixed in [v1.10.4](https://github.com/dinbtechit/vscode-theme/releases/tag/v1.10.4)
